tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -4646,150 +4677,105 @@ TEST_F(OpConverterTest, ConvertPool3D) {
     std::vector<int> strides;
     string padding;
     string data_format;
-    bool is_max_pooling;
     std::vector<int> expected_output_dims;
-    std::vector<float> expected_output;
+    // output vector: {MaxPool2D, AvgPool2D, MaxPool3D, AvgPool3D }
```

Comment:
Let's add an "s" to the variable name and modify the comment to:
// The expected outputs for the following operations being tests: MaxPool2D, AvgPool2D, MaxPool3D, AvgPool3D

Reply:
done